### PR TITLE
bucket retrieving by 3 times

### PR DESCRIPTION
**Pull request by stingion**

### CODE REVIEW

File: scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java [hunk 1]
```diff
@@ -20,10 +24,21 @@ public CouchbaseRepositoryFactory(CouchbaseSettings settings) {
 
     Cluster cluster = nodes.isEmpty() ? CouchbaseCluster.create() : CouchbaseCluster.create(nodes);
 
-    bucket =
-        cluster
+    int time = 0;
+    int maxTimes = 3;
+    do {
```
Comment (dmytro-lazebnyi) [inline scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java]: Maybe let's use Reactor's `retryBackoff`?

Comment (stingion) [reply scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java]: Done

File: scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java [hunk 2]
```diff
@@ -20,10 +22,11 @@ public CouchbaseRepositoryFactory(CouchbaseSettings settings) {
 
     Cluster cluster = nodes.isEmpty() ? CouchbaseCluster.create() : CouchbaseCluster.create(nodes);
 
-    bucket =
+    bucket = Mono.fromCallable(() ->
         cluster
             .authenticate(settings.username(), settings.password())
-            .openBucket(settings.organizationsBucketName());
+            .openBucket(settings.organizationsBucketName())).retryBackoff(3, Duration.ofSeconds(1))
```
Comment (segabriel) [inline scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java]: I like infinity, why do you afraid of this endless?

Comment (stingion) [reply scalecube-organization/src/main/java/io/scalecube/organization/repository/couchbase/CouchbaseRepositoryFactory.java]: It's discussed solution, I you can explain why it's better to use another amount of retries, we can consider it.

**Pull request merged**